### PR TITLE
Add runbooks for API response times alarms

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Related to #2501 by @sarayourfriend

## Description

Adds the run books for the general API response time alarms.

## Testing Instructions
Look for the preview in the comments and check that the instructions make sense.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by dhruvkb — APPROVED** [review]
Looks like both of these runbooks are largely identical in content (as they should be, they're both for response times and likely to co-occur frequently). Can they both be combined into a single runbook instead?

That's an infra change, which might delay things, so I'm approving this to unblock. LGTM!

**Review by sarayourfriend — APPROVED** [review]

File: documentation/meta/monitoring/runbooks/api_avg_response_time_above_threshold.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Run Book: API Production Average Response Time above threshold
+
+```{admonition} Metadata
+Status: **Unstable**
+
+Maintainer: @krysaldb
+
+Alarm link:
+- <https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#alarmsV2:alarm/API+Production+Average+Response+Time+under+threshold?>
```
Comment (obulat) [inline documentation/meta/monitoring/runbooks/api_avg_response_time_above_threshold.md:9]:
```suggestion
- <https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#alarmsV2:alarm/API+Production+Average+Response+Time+above+threshold?>
```

File: documentation/meta/monitoring/runbooks/api_p99_response_time_above_threshold.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Run Book: API Production Average Response Time above threshold
+
+```{admonition} Metadata
+Status: **Unstable**
+
+Maintainer: @krysaldb
+
+Alarm link:
+- <https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#alarmsV2:alarm/API+Production+Average+Response+Time+under+threshold?>
```
Comment (obulat) [inline documentation/meta/monitoring/runbooks/api_p99_response_time_above_threshold.md:9]:
```suggestion
- <https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#alarmsV2:alarm/API+Production+P99+Response+Time+above+threshold?>
```

File: documentation/meta/monitoring/runbooks/api_p99_response_time_above_threshold.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Run Book: API Production Average Response Time above threshold
```
Comment (obulat) [inline documentation/meta/monitoring/runbooks/api_p99_response_time_above_threshold.md:1]:
```suggestion
# Run Book: API Production P99 Response Time above threshold
```
Should it be p99?

**Review by obulat — CHANGES_REQUESTED** [review]
The links should be updated, and probably the heading.

File: documentation/meta/monitoring/runbooks/api_avg_response_time_above_threshold.md (hunk 1 above)
Comment (krysal) [reply documentation/meta/monitoring/runbooks/api_avg_response_time_above_threshold.md:9]: Good catch, thanks!

**Comment by krysal** [comment]
@obulat I added the runbooks for the anomaly response alarms (WordPress/openverse-infrastructure#622), I'll await your review on this.

**Review by obulat — APPROVED** [review]

**Pull request merged**